libs/storage/Tsavorite/cs/src/core/Allocator/BlittableAllocator.cs

```diff
@@ -17,10 +17,10 @@ internal sealed unsafe class BlittableAllocator<Key, Value> : AllocatorBase<Key,
         private readonly long* nativePointers;
 
         // Record sizes
-        private static readonly int recordSize = Utility.GetSize(default(Record<Key, Value>));
-        private static readonly int recordInfoSize = Utility.GetSize(default(RecordInfo));
-        private static readonly int keySize = Utility.GetSize(default(Key));
-        private static readonly int valueSize = Utility.GetSize(default(Value));
+        private static readonly int recordSize = Unsafe.SizeOf<Record<Key, Value>>();
```

Comment:
It is unnecessary to cache these values in statics. `Unsafe.SizeOf` evaluates to constant, so it is cheaper to call it inline or to wrap it in a property.

Reply:
is this true for .NET 6 as well?

Reply:
Yes, it is true for all .NET versions.